internal/pkg/agent/install/uninstall.go

```diff
@@ -97,8 +97,8 @@ func Uninstall(cfgFile, topPath, uninstallToken string) error {
 // to an ERROR_SHARING_VIOLATION. RemovePath will retry up to 2
 // seconds if it keeps getting that error.
 func RemovePath(path string) error {
-	const arbitraryTimeout = 2 * time.Second
-	var start time.Time
+	const arbitraryTimeout = 5 * time.Second
```

Comment:
Sorry to be "that guy" but why are we spinning in this for loop instead of sleeping between iterations?

Reading the code I thought we would sleep between attempts until we would exceed the timeout...